handsontable/src/plugins/collapsibleColumns/collapsibleColumns.js

```diff
@@ -544,7 +550,10 @@ export class CollapsibleColumns extends BasePlugin {
 
         // Add ARIA tags
         if (isAriaTagsEnabled) {
-          setAttribute(TH, ...A11Y_EXPANDED(false));
+          setAttribute(TH, [
+            A11Y_EXPANDED(false),
+            A11Y_DESCRIPTION(this.hot.getTranslatedPhrase(COLUMN_HEADER_DESCRIPTION_EXPAND_COLUMN)),
```

Comment:
The expand/collapse button is hidden from the screen readers, so I added an instruction how to expand/collapse the column,